core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -141,8 +143,8 @@ private[spark] class HighlyCompressedMapStatus private (
     private[this] var numNonEmptyBlocks: Int,
     private[this] var emptyBlocks: RoaringBitmap,
     private[this] var avgSize: Long,
-    @transient private var hugeBlockSizes: Map[Int, Byte])
-  extends MapStatus with Externalizable {
+    @transient private[this] var hugeBlockSizes: Map[Int, Byte])
+  extends MapStatus with Externalizable with KryoSerializable {
```

Comment:
OK, I have manual tests `remove @transient` and `Extends KryoSerializable`, both worked fine,  I'm doing UT.